app/models/agents/trello_notification_agent.rb

```diff
@@ -0,0 +1,161 @@
+require 'trello'
+
+module Agents
+  class TrelloNotificationAgent < Agent
+    no_bulk_receive!
+    default_schedule "every_1h"
+    cannot_receive_events!
+    gem_dependency_check { defined?(Trello) }
+
+    description <<-MD
+      The Trello Notification agent allows you to get notifications from your Trello account.
+
+      To use the Trello Notification agent you will first need to sign into your Trello account.
+      Once you have signed in go to the following url [https://trello.com/app-key](https://trello.com/app-key) and copy the key.
+      This will be used as the agent `public_key`.
+
+      Next on the same page click on the `Token` link and copy the token displayed. This will be used as the agent `member_token`.
+
+      Additionally you can specify the Trello board name with the `board_name` option. This will restrict the agent to notifications of that particular board. If no board is specified, then notifications from all boards is returned.
+
+      You will also need to specify you Trello username as `username`. If no username is specified it will get a list of notifications for all members of the specified board.
+
+      Finally you will also need to specify the max age of the notifications to retrieve with the `max_age` option. By default this is 0 which will retrieve all notifications. 1d retrieves all notifications 1 day old, 2d - 2 days old, 1h - 1 hour old, 1m - 1 minute old etc.
+    MD
+
+    event_description <<-MD
+      Here's an example of the event data that is returned by the Trello notifications agent. This is just one notification however more data could be returned based on your configuration. Different types of data can also be returned depending on the type of notification.
+
+          {
+            "notifications":
+            {
+              "id": "1119afa1ggg775cg2fmom9a0",
+              "unread": false,
+              "type": "commentCard",
+              "date": "2016-06-17T08:06:57.843Z",
+              "data": {
+                "text": "@user ready for testing.",
+                "card": {
+                  "shortLink": "bC78glii",
+                  "idShort": 200,
+                  "name": "As a user, when I'm on the Edit Profile Page, if I edit the form and hit save, it should close the form.",
+                  "id": "573c735c13dcb2411a32d6bd"
+                },
+                "board": {
+                  "shortLink": "H1RadcyM",
+                  "name": "Software App",
+                  "id": "54b573g3454906237bc9cb70"
+                }
+              },
+              "member_creator_id": "4eg0df123bf061922g054540"
+            }
+          }
+    MD
+
+    def configure
```

Comment:
Again same as above, I just configure the API in every check call. Not sure if there's a better way.